### DISCUSSION TITLE
Exit fullscreen

### DISCUSSION THREAD

**Discussion by edweerdt** [AppGround-io/bluetooth-keyboard-and-mouse-support#276] [Bugs & Problems]
I might be daft, but how do you exit fullscreen and get back to the app menu/settings? Pressing the back button gives me the options cancel and disconnect& quit in a pop-up. I can see the rest of the app in the background but there is no way to interact with it.

S21Ultra

**Comment by rockymehta1** [comment]
Same issue. Swiping up and down doesn't work for me. Samsung Galaxy 22+ Ultra Android 13

**Comment by taaglab** [comment]
Same issue on A53 5G Android 13, however in a pinch I've found you can open in pop up view to change modes etc.

**Comment by AppGround-io** [comment]
Thank you for the reports. It seems to only affect Samsung devices running Android 13, the fix for it will be available in the new version 4.23.1 in the next few days.
